cmd/debos/debos.go

```diff
@@ -265,7 +265,12 @@ func main() {
 			exitcode = 1
 			return
 		}
-		m.SetMemory(int(memsize / 1024 / 1024))
+
+		memsizeMB := int(memsize / 1024 / 1024)
+		if memsizeMB < 2048 {
+			log.Printf("WARNING: Memory size of %dMB is less than recommended 2048MB\n", memsizeMB)
```

Comment:
Fwiw less then  2G is fine; It's probably when it's less then say 512m or even 256 i'd start really questioning  things.

Reply:
changed it to say "recommended minimum" and set that limit to be 256M